src/__tests__/index.js

```diff
@@ -152,6 +165,12 @@ test(
     4
 );
 
+test('should handle background value shorthand',
+    assertColourStops,
+    `header{background:resemble-image(url("${image}")) no-repeat center / cover}`,
```

Comment:
Seems like a regex is a more robust choice if we don't want to hardcode the index in the test. Updated with the next commit.